client/components/advanced-credentials/host-info.ts

```diff
@@ -16,23 +15,23 @@ enum InfoTypes {
 
 export interface Text {
 	type: InfoTypes.Text;
-	text: ReactChild;
+	text: string;
```

Comment:
You don't want to do any changes in this file. The `ReactChild` type is correct, as the text can be a React markup (with HTML), not just a string. And there shouldn't be any `translate( ... ).toString()`, just `translate()`.